### ISSUE TITLE
Monokai

### ISSUE THREAD

**Issue by colinxjj** [spyder-ide/spyder#6861]
## Description

**What steps will reproduce the problem?**

1. 
2. 
3. 

**What is the expected output? What do you see instead?**

**Please provide any additional information below**

  File "D:\Program Files\Anaconda3\lib\site-packages\spyder\plugins\configdialog.py", line 214, in accept
    configpage.apply_changes()
  File "D:\Program Files\Anaconda3\lib\site-packages\spyder\plugins\configdialog.py", line 104, in apply_changes
    self.apply_callback()
  File "D:\Program Files\Anaconda3\lib\site-packages\spyder\plugins\configdialog.py", line 269, in <lambda>
    self.apply_settings(self.changed_options))
  File "D:\Program Files\Anaconda3\lib\site-packages\spyder\plugins\configdialog.py", line 1184, in apply_settings
    self.main.ipyconsole.apply_plugin_settings(['color_scheme_name'])
  File "D:\Program Files\Anaconda3\lib\site-packages\spyder\plugins\ipythonconsole.py", line 710, in apply_plugin_settings
    client.set_color_scheme(color_scheme_o)
  File "D:\Program Files\Anaconda3\lib\site-packages\spyder\widgets\ipythonconsole\client.py", line 437, in set_color_scheme
    self.shellwidget.set_color_scheme(color_scheme)
  File "D:\Program Files\Anaconda3\lib\site-packages\spyder\widgets\ipythonconsole\shell.py", line 143, in set_color_scheme
    self.reset(clear=True)
  File "D:\Program Files\Anaconda3\lib\site-packages\qtconsole\frontend_widget.py", line 665, in reset
    self._show_interpreter_prompt()
  File "D:\Program Files\Anaconda3\lib\site-packages\qtconsole\jupyter_widget.py", line 348, in _show_interpreter_prompt
    msg_id = self.kernel_client.execute('', silent=True)
AttributeError: 'NoneType' object has no attribute 'execute'

## Version and main components

* Spyder Version: 3.2.6 
* Python Version: 3.6.4
* Qt Versions: 5.6.2, PyQt5 5.6 on Windows

## Dependencies
```
pyflakes >=0.6.0 :  1.6.0 (OK)
pycodestyle >=2.3:  2.3.1 (OK)
pygments >=2.0   :  2.2.0 (OK)
pandas >=0.13.1  :  0.22.0 (OK)
numpy >=1.7      :  1.14.0 (OK)
sphinx >=0.6.6   :  1.6.6 (OK)
rope >=0.9.4     :  0.10.7 (OK)
jedi >=0.9.0     :  0.11.1 (OK)
nbconvert >=4.0  :  5.3.1 (OK)
sympy >=0.7.3    :  1.1.1 (OK)
cython >=0.21    :  0.27.3 (OK)
qtconsole >=4.2.0:  4.3.1 (OK)
IPython >=4.0    :  6.2.1 (OK)
pylint >=0.25    :  1.8.2 (OK)

```

**Comment by ccordoba12** [comment]
This problem is fixed in our latest version (3.2.8). Please update by opening the Anaconda prompt and running there

    conda update spyder

**Comment by CAM-Gerlach** [comment]
This error is a duplicate of #6519 , which is fixed with Spyder 3.2.8.
